mealplanner-ui/src/pages/Meals/MealTags.tsx

```diff
@@ -0,0 +1,65 @@
+import { Chip, Grid, Stack } from "@mui/material";
+import { graphql } from "babel-plugin-relay/macro";
+import { useFragment } from "react-relay";
+import { MealTags_tags$key } from "./__generated__/MealTags_tags.graphql";
+import { setSelectedMealTags } from "../../state/state";
+
+export const MealTagsFragment = graphql`
+  fragment MealTags_tags on Query
+    @refetchable(queryName: "MealTagsRefetchQuery") {
+        gqLocalState {
+            selectedMealTags
+        }
+        allMealTags(first:100) {
+            edges {
+                node
+            }
+        }
+  }
+`;
+
+export const MealTags = ({ tags }: { tags: MealTags_tags$key }) => {
+  const mealTags = useFragment(MealTagsFragment, tags);
+  // If nothing is selected, mealTags.gqLocalState.selectedMealTags will return a null
+  // When it returns a null, we need to assign an empty array.
+  const selectedMTags = mealTags.gqLocalState.selectedMealTags || [];
```

Comment:
Let's use full English names: selectedMealTags instead of the cryptic selectedMTags.

Reply:
It is not cryptic. It is pretty clear that gqLocalState.selectedMealTags is assigned to selectedMTags. I had used it as selectedMPTags. So she had used the same convention.